app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt

```diff
@@ -5,6 +5,9 @@ import android.content.Intent
 import android.os.Bundle
 import android.support.v4.content.ContextCompat
 import android.support.v7.app.AppCompatActivity
+import android.text.Editable
+import android.text.TextWatcher
+import android.util.Log
```

Comment:
Where is this being used?

Reply:
@arundhati24 I have remove Log, Rest Editable and TextWatcher are used at line 134, 143.